CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.3)
 
 project(EDM4HEP VERSION 0.1.0)
 
+#--- User-definable options --------------
+
+option(BUILD_DDG4EDM4hep "Build the EDM4hep plugin for DD4hep" ON)
```

Comment:
I think the cmake variables should be all uppercase for reasons of sanity and consistency

Reply:
Done